linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/summary.js

```diff
@@ -226,21 +240,22 @@ var Interfaces = (function() {
   }
 
   function renderInterfaces(routers, namers, template) {
-    var servers = _(routers)
-      .map('servers')
-      .flatten()
-      .value();
+    var namerIfaces = _.map(namers, prepInterface);
 
-    var clients = _(routers)
-      .map(function(router) { return _.values(router.dstIds); })
-      .flatten()
-      .value();
+    $('#client-info').html(template({
+      name:'router clients',
+      interfaces: prepClients(routers.clients)
```

Comment:
When I run the admin server with your changes, the `prepClients` method throws the following exception:

    Uncaught TypeError: Cannot read property 'map' of undefined

I think that means that calling routers.clients here returns undefined, but I don't see anything obvious in that method definition that would be causing this.